discover/portal/articles/210-setting-up/02-instance-settings/00-intro.markdown

```diff
@@ -0,0 +1,24 @@
+# Setting Up a Virtual Instance [](id=setting-up-a-virtual-instance)
+
+Once @product@ is installed, the configuration begins. Whether that means adding
+another virtual instance or setting up the initial instance to suit your needs.
+Recall that configuration happens at different scopes. Here were thinking about
```

Comment:
Should "were" be "we are". 
Also I'm not sure if "thinking" is the most appropriate verb, but I'll leave it up to you.